src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -24,34 +29,28 @@ Copyright (c) .NET Foundation. All rights reserved.
     <SuppressDependenciesWhenPacking Condition=" '$(PackAsTool)' == 'true' ">true</SuppressDependenciesWhenPacking>
   </PropertyGroup>
 
-  <Target Name="PackTool" DependsOnTargets="GenerateToolsSettingsFileFromBuildProperty;Publish;ResolveApphostAsset" Condition=" '$(PackAsTool)' == 'true' ">
-
-    <NETSdkError Condition=" '$(SelfContained)' == 'true' "
-                 ResourceName="PackAsToolCannotSupportSelfContained" />
-
-    <NETSdkError Condition=" '$(TargetFrameworkIdentifier)' != '.NETCoreApp' "
-                 ResourceName="DotnetToolOnlySupportNetcoreapp" />
-
-    <NETSdkError Condition=" '$(_TargetFrameworkVersionWithoutV)' &lt; '2.1' "
-                 ResourceName="DotnetToolDoesNotSupportTFMLowerThanNetcoreapp21" />
-
+  <Target Name="PackTool" DependsOnTargets="GenerateToolsSettingsFileFromBuildProperty;Publish;_PackToolValidation" Condition=" '$(PackAsTool)' == 'true' ">
     <ItemGroup>
       <_GeneratedFiles Include="$(PublishDepsFilePath)"/>
       <_GeneratedFiles Include="$(PublishRuntimeConfigFilePath)"/>
       <_GeneratedFiles Include="$(_ToolsSettingsFilePath)"/>
     </ItemGroup>
 
+    <RelativePathFormatInPublishToFormatInPackPackage
```

Comment:
This name is still confusing me. Can it be something like `ResolveToolPackagePaths`?